man/mount.efs.8

```diff
@@ -87,6 +99,20 @@ Mount an EFS file system with file system ID "fs-abcd1234" at mount point \
 sudo mount -t efs -o tls,verify=0 fs-abcd1234 /mnt/efs
 Mount an EFS file system with file system ID "fs-abcd1234" at mount point \
 "/mnt/efs" using encryption of data in transit and a verify level of 0\&.
+.TP
+sudo mount -t efs fs-abcd1234.efs.us-east-1.amazonaws.com /mnt/efs
+Mount an EFS file system with the fully-qualified EFS DNS name \
+"fs\-abcd1234\&.efs\&.us-east-1\&.amazonaws\&.com" at mount point \
+"/mnt/efs" without encryption of data in transit\&. If the region \
```

Comment:
Yes: since the CNAME is resolved to just the FS-ID, everything that works if you mount by FS-ID also works if you mount by CNAME -- including TLS!

I didn't want to simply replicate all the examples, but I did add the TLS-one to make it clear that this will work!